extensions/amp-lightbox-gallery/amp-lightbox-gallery.md

```diff
@@ -163,6 +163,11 @@ If no thumbnail is specified, `<amp-img>` elements will be cropped per
 `poster` attribute, and placeholder images will be used for lightboxed elements
 that have one.
 
+## Styling
+
+You can style the `amp-lightbox-gallery-caption` component with standard CSS. Some of the
```

Comment:
Perhaps this should indicate the element has a class value equal to this (and isn't a component).

Reply:
Sounds good! I just updated the description and added a few lines to the corresponding test file.